vignettes/sidebars.Rmd

```diff
@@ -1,38 +1,50 @@
 ---
 title: "Sidebars"
+output:
+  html_document:
+    theme:
+      version: 5
```

Comment:
Is this needed? Or just for your own development? FWIW, I use something like `nodemon --exec ./build.sh --watch vignettes --ext Rmd` (with `build.sh` containing something like `R -e "pkgdown::build_article('sidebars')"`

(If you have a better "hot reloading" setup feel free to let me know :))

```suggestion
```